CMakeLists.txt

```diff
@@ -55,33 +62,51 @@ target_link_libraries(sxg
 )
 
 ########################################
-## Testing
+## Installing
 ########################################
 
-enable_testing()
-file(COPY tests/testdata DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
-
-ExternalProject_Add(
-  gtest
-  GIT_REPOSITORY https://github.com/google/googletest
-  GIT_TAG master
-  SOURCE_DIR ${PROJECT_BINARY_DIR}/third_party/gtest
-  BINARY_DIR ${PROJECT_BINARY_DIR}/gtest
-  INSTALL_COMMAND ""
+set_target_properties(sxg PROPERTIES PUBLIC_HEADER "${HEADERS}")
+INSTALL(TARGETS sxg
```

Comment:
It looks like you sometimes write the first argument right after the open parenthesis, and sometimes in a separate line. What is the rule of whether or not to break the line?

Reply:
OK, uniform it to break the line.